sdk/table/azure-table/dev_requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e ../../../tools/azure-devtools
+-e ../../../tools/azure-sdk-tools
+../../core/azure-core
+-e ../../identity/azure-identity
```

Comment:
If I remember correctly, AAD auth is not supported. we can remove the azure-identity